src/client/app/filter/FilterIndex.tsx

```diff
@@ -149,17 +168,17 @@ const FilterIndex = ({ initialQuery, lagre, avbryt, tittel, visningV3, køvisnin
 			.then((dataRes) => {
 				if (dataRes.payload !== REQUEST_POLLING_CANCELLED) {
 					setOppgaver(updateIdentities(dataRes.payload));
-					setQueryError(null);
+					setQueryErrorMessage(null);
 					setLoading(false);
 				} else {
 					setOppgaver([]);
-					setQueryError('Klarte ikke å kjøre søk grunnet tidsavbrudd.');
+					setQueryErrorMessage('Klarte ikke å kjøre søk grunnet tidsavbrudd.');
 					setLoading(false);
 				}
 			})
 			.catch(() => {
 				setOppgaver([]);
-				setQueryError('Klarte ikke å kjøre søk grunnet ukjent feil.');
+				setQueryErrorMessage('Klarte ikke å kjøre søk grunnet ukjent feil.');
```

Comment:
Dette er vel ikkje endra i denne PR, så mest til ettertanke, men burde ein vurdere å rethrowe feilen istadenfor å sluke den her, sidan ein vel fanger alle mulige feil slik koden står her? 

Eller er det noko som logger feil med info tidlegare i systemet? Eg kjenner ikkje los, men blir vel i alle fall vanskeleg å debugge ting viss ein ikkje har noko meir feilinfo enn dette.

Reply:
du har helt rett i det. Jeg har laget en oppgave på å sette opp feilhåndtering på api-kall som viser feilmelding under navbaren, for det er en mangel i appen generelt nå. 

Grunnen til at jeg legger inn frontend-validering er fordi responsen fra backend gjør det vanskelig å utlede hvor feilen i det brukeren sender inn er.